Nos/Service/Relay/RelayService.swift

```diff
@@ -399,6 +400,9 @@ extension RelayService {
                     "\(remainingEventCount) events left in parse queue."
                 )
                 #endif
+                if remainingEventCount >= 1000 && remainingEventCount < 1030 {
+                    self.crashReporting.report("Parse queue is large: currently \(remainingEventCount) events")
```

Comment:
I'm worried about how this will show up in Sentry, but it isn't easy to test... I want to make sure all the events show up under the same Sentry issue, and I'm worried if the `remainingEventCount` is different every time then they may show up as separate issues.

Since we know the `remainingEventCount` is between 1000 and 1030 can we just change the message to "current over 1000 events" or similar?

Reply:
Ah, yeah, I was able to trigger it to happen a couple times -- I'm just not sure where to find it in Sentry. I'm happy to look and see what happened with my two (or more) reports, but I'm also happy to change this.

Reply:
I made a change to ensure it's always the same.

Reply:
It won't show up in Sentry unless you do a release build with the sentry API key in your secrets file :/ But I'm good to ship this and maybe you can check out what it looks like on staging.

Reply:
I tested on staging, tried my best to reproduce the error, and was able to it in Sentry. It works!

<img width="697" alt="Screenshot 2024-07-31 at 11 38 33 AM" src="https://github.com/user-attachments/assets/af34e2e8-06e2-43d0-bcc5-a28741191f8b">